### ISSUE TITLE
Can't Open BR Generated Missions in DCS - (cause: using parked on airbase for non aircraft)

### ISSUE THREAD

**Issue by GrEaSeLiTeNiN** [akaAgar/briefing-room-for-dcs#390]
### Brief Description: 
1. Generated missions sometimes have errors (top right yellow prompt indicator that requires clearing).
2. Some missions cannot be opened in DCS ('red' error message) and after that happens, DCS cannot open any mission file in Mission or Mission Editor unless the BR mission file is deleted.
3. It's as if BR sometimes generates 'corrupted' mission files that disable DCS Mission or Mission Editor. 
4. Reset Template to Default works again. Some mission option combinations may be causing loading errors.

### Expected: 
Previous BR version was more stable and did not have the above issues.
 
#### Mission and Template Zip: 
See attached. 
[BR Missions Cannot Load.zip](https://github.com/akaAgar/briefing-room-for-dcs/files/8184628/BR.Missions.Cannot.Load.zip)

Mission modified: 
No

**Comment by GrEaSeLiTeNiN** [comment]
I tried the mission planning options one by one and think I found the problem -> My targets' behavior have to be set to "Idle" and not "Parked on Airbase (recommend disable task)".

**Comment by john681611** [comment]
Yeah parked on airbase is for aircraft only unfortunatly. We should protect against that kinda stuff
